src/modules/tiers/selectors.ts

```diff
@@ -0,0 +1,33 @@
+import { LoadingState } from 'decentraland-dapps/dist/modules/loading/reducer'
+import { isLoadingType } from 'decentraland-dapps/dist/modules/loading/selectors'
+import { RootState } from 'modules/common/types'
+import { getPendingTransactions } from 'modules/transaction/selectors'
+import { FETCH_THIRD_PARTY_ITEM_TIERS_REQUEST, BUY_THIRD_PARTY_ITEM_TIERS_SUCCESS, BUY_THIRD_PARTY_ITEM_TIERS_REQUEST } from './actions'
+import { ThirdPartyItemTier, TiersState } from './types'
+
+function getState(state: RootState): TiersState {
```

Comment:
I don't really mind this, but if you want, we could change this file to use constants
```ts
export const getState (...)
```

so it looks like all the other selectors in the codebase/other dapps.

Following that path, we can keep consistent and add the four selectors we have in all files:

```ts
export const getState = (state: RootState) => state.item
export const getData = (state: RootState) => getState(state).data
export const getLoading = (state: RootState) => getState(state).loading
export const getError = (state: RootState) => getState(state).error
```

which'd allow for not having to do `.data` below too

Reply:
We're using lambdas in all of our other selectors, so I don't mind at all! Changed.